schemas/expconf/v0/searcher-adaptive-asha.json

```diff
@@ -33,6 +33,20 @@
             "default": null,
             "minimum": 1
         },
+        "time_metric": {
+            "type": [
+                "string",
+                "null"
+            ],
+            "default": null
+        },
+        "max_time": {
```

Comment:
also, shouldn't async_halving also be updated to include time_metric and max_time?

Reply:
see other comment: i thought `async_halving` was a deprecated searcher.